python/aqora_cli/qiskit/client.py

```
@@ -0,0 +1,243 @@
+    if scheme not in ("https", "http"):
+        raise ValueError(f"URL must use http or https scheme: {url!r}")
+
+
+def _put_json_payload(upload_url: str, payload: str) -> str:
+    _require_http_scheme(upload_url)
+    request = urllib.request.Request(
+        upload_url,
+        data=payload.encode("utf-8"),
+        method="PUT",
+        headers={"Content-Type": "application/json"},
+    )
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)